### ISSUE TITLE
SA1100 does not support inherited indexers

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1247]
In the following code, SA1100 should be reported on the keyword `base`.

```csharp
class ClassName : List<int>
{
  public int Property { get { return base[0]; } }
}
```

**Comment by pdelvo** [comment]
grabbed
